falkordb-node/node-entrypoint.sh

```
@@ -313,6 +313,36 @@
+
+config_rewrite() {
+  # Config rewrite
+  echo "Rewriting config"
+  redis-cli -p $NODE_PORT -a $ADMIN_PASSWORD --no-auth-warning $TLS_CONNECTION_STRING CONFIG REWRITE
+}
+
+if [ -f $NODE_CONF_FILE ]; then
+  # Get current admin password
+  CURRENT_ADMIN_PASSWORD=$(cat $NODE_CONF_FILE | grep -oP '(?<=requirepass ).*')
+  # If current admin password is different from the new one, reset it
+  if [[ $CURRENT_ADMIN_PASSWORD != $ADMIN_PASSWORD ]]; then
```

Shellcheck

[warning] Quote the right-hand side of != in [[ ]] to prevent glob matching.

(SC2053)